schemas/blackmarket-README.md

```diff
@@ -0,0 +1,56 @@
+# EDDN BlackMarket Schema
+
+## Introduction
+
+**This schema is deprecated.  The same data has been made available via the
+`prohibited` array in the [commodity schema](./commodity-v3.0.json) since 
+September 2017.**
+
+What follows below is only for historical curiosity.
+
+---
+
+Here we document how to take data from an ED `BlackMArket` Journal Event and
+properly structure it for sending to EDDN.
+
+Please consult [EDDN Schemas README](./README-EDDN-schemas.md) for general
+documentation for a schema such as this.
+
+## Senders
+The primary data source for this schema is the ED Journal event `MarketSell`.
+
+### Key Renames
+#### name
+Due to how the EDDN schema is defined the `Type` key/value should
+have the key renamed to `name`.
+
+#### prohibited
+Due to how the EDDN schema is defined the `IllegalGoods` key/value should
+have the key renamed to `prohibited`.
+
+#### marketID
+The Journal documentation says this is `MarketID`, but in the schema the 
+`m` is lower case.
+
+### Elisions
+You MUST remove the following key/value pairs from the data:
+
+  - `Count`
```

Comment:
`Count` may be potentially useful in some cases. Historically, tracking UA bombing for example.

Reply:
True, but really I think we're aiming at removing this schema entirely.

I *think* I based this elision on what the removed EDMC code did back in 2017, as it was the only reference I could find.

Reply:
Ah, the EDMC code doesn't *include* it, which I took to imply it should be removed, but that's not true.  Removed from that section of the docs.